### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by FerFemNemBem** [naz664/crDroid_gsi#19] [Q&A] [answered]
Hi,

I'm using this nice ROM on my Redmi Note 10T 5G (because there is no official crDroid release for "camellian") and it is - and works great!
How can I update the ROM to the most recent version without loosing data? (currently I'm on 9.3 and want to go to 9.5)
adb sideload needs a zip - not an image.

@naz664 - any idea about the best way to stay on the most recent version of this ROM?

Cheers,
FFNB

I'ts not my daily driver so if things happen, they happen. ;)

**Comment by naz664** [comment, marked as answer]
Well you can follow the same instructions like you did for flashing all other gsi's.
And i recommend to backup all data before dirty flashing incase if something bad happen that would help.

**Reply by FerFemNemBem** [reply]
Hi,

I'm new to all this... But I followed your recommendation and indeed it was as simple as:
```
fastboot reboot fastboot
fastboot flash system s:\tmp\crDroid-9.5-arm64_bgN-slim-Unofficial.img
fastboot reboot
```
to update the device.
Thank you!
Cheers,
FFNB

**Reply by naz664** [reply]
Ok cool
